### DISCUSSION TITLE
Dependabot-Core licence

### DISCUSSION THREAD

**Discussion by damiang78** [community/community#18649] [Other Feature Feedback, Questions, & Ideas]
Hi, 

We would like to host an instance of Dependabot with GitLab privately at our workplace for the use of updating our project dependencies as well as identifying security vulnerabilities. Could you please clarify if the The Prosperity Public License 2.0.0 allows us to do this freely? 

Thanks
Damian

**Comment by 5HT2** [comment]
You would be better off asking in the Dependabot repository, or your workplace's legal team.

> This license lets you use and share this software for free,
with a trial-length time limit on commercial use. Specifically:

The top of the license has this, which means that you would likely need special permission to use it commercially.

**Reply by damiang78** [reply]
Thanks will ask in the repository
